vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.cpp

```diff
@@ -853,7 +853,10 @@ std::string WindowsTextInputComponentView::GetTextFromRichEdit() const noexcept
 
 void WindowsTextInputComponentView::finalizeUpdates(RNComponentViewUpdateMask updateMask) noexcept {
   // m_element.FinalizeProperties();
```

Comment:
unrelated to your change, but while you're here, considering deleting this seemingly forgotten comment